ray-operator/controllers/ray/raycluster_controller.go

```diff
@@ -337,6 +337,15 @@ func (r *RayClusterReconciler) rayClusterReconcile(ctx context.Context, request
 		}
 		return ctrl.Result{RequeueAfter: DefaultRequeueDuration}, err
 	}
+	// only reconcile serve service is "ray.io/enableAgentService" annotation is set to true
+	if enableAgentServiceValue, exist := instance.Annotations[common.EnableAgentServiceKey]; exist && enableAgentServiceValue == common.EnableAgentServiceTrue {
```

Comment:
`EnableAgentServiceKey` -> `EnableServeServiceKey`

Reply:
Sorry,  `EnableAgentServiceKey` is the only one defined